Lib/test/libregrtest/runtest.py

```diff
@@ -352,12 +369,11 @@ def _runtest_inner2(ns: Namespace, test_name: str) -> bool:
 
     support.reap_children()
 
-    return refleak
+    return result
 
 
-def _runtest_inner(
-    ns: Namespace, test_name: str, display_failure: bool = True
-) -> TestResult:
+def _runtest_inner(result: TestResult, ns: Namespace,
+                   display_failure: bool = True):
```

Comment:
Could you add a return type annotation here, like there is in the function you're replacing? Most of the functions in this file have complete type annotations